pkg/config/vault.go

```diff
@@ -506,3 +507,21 @@ func toStringSlice(interfaceSlice []interface{}) []string {
 	}
 	return retSlice
 }
+
+func initializeOIDCToken(client VaultClient, hookConfig map[string]interface{}) error {
```

Comment:
Should we refactor the way that the token is set as env var? Right now it takes a moment to realize that "initialize" means setting it in the env.

Perhaps a separate function in `pkg/vault/oidc.go` that exposes the token in to the environment, that you call in `initializeOIDCToken()`. Or a comment on top of this function would do to.